jep/0000/README.adoc

```diff
@@ -0,0 +1,315 @@
+
+==== hacktoberfest
+
+Issues labelled `hacktoberfest` are labelled by the Hacktoberfest project and will help new contributors find issues that they will like to work on.
+
+==== GitHub CLI
+
+GitHub has a really powerful CLI that can be used to automate management, retrieve issues assigned to the user or with a certain label and update the issue.
+All without having to visit the browser at all.
+
+==== Issue bounties
+
+It would enable issue bounty systems that integrate with GitHub as requested in the link:https://groups.google.com/g/jenkinsci-dev/c/Z2PLVBEGar0/m/pjlesVANBAAJ[mailing list].
+
+== Reasoning
+
+=== Features we can take advantage of
+
+==== Bots
+
+With GitHub actions we can easily integrate additional automated features into our workflow.
+
+Here are some ideas for features that we can add that we don't have in Jira:
+- Newcomer bot
+- Saved replies (aka canned responses)
+- Automatic labelling based on criteria
+- Automatic responses based on labels, e.g. priority
+- New issues can be validated before being accepted by adding `needs-triage` label
+
+==== Workflows and issue templates as code
+
+We will be able to define issue templates as code using link:https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms[GitHub issue forms].
+
+This will allow easy iteration on fields, help text, and workflow. Opening up new possibilities that were never attempted on Jira due to the difficulty in changing the forms.
+
+See link:https://github.com/jenkins-infra/helpdesk/issues/new/choose[jenkins-infra/helpdesk] for example forms that the Infra team uses.
+
+==== No reliance on infra team or Linux foundation
+
+As GitHub is a SaaS it will be automatically updated and managed for us.
+
+While linux foundation runs Jira for us there is still some cost involved to the Infra team and developer community. There's downtime when it's patched and configuration changes are generally done by the Infra team.
+
+==== Triage party
+
+link:https://github.com/google/triage-party[Triage party] is available to us to improve our issue triage and management of issues.
+
+Bringing features like:
+- Highlighting latency by project members (responded: +15d)
+- Comment popularity (comments-per-month: >0.9)
+- 'Multi-player triage' have multiple people triaging together splitting the issues
+
+=== Equivalent capability
+
+==== Duplicate issues
+
+On Jira we used a plugin to detect potential duplicate issues
+On GitHub we will use link:https://github.com/actions-cool/issues-similarity-analysis[actions-cool/issues-similarity-analysis] to do the same thing as used on the link:https://github.com/jenkins-infra/helpdesk/blob/main/.github/workflows/issues-similarity.yml[Infra helpdesk].
+
+==== Dashboards
+ 
+A number of Jira dashboards are setup.
```

Comment:
I don't know if it's the case for Jira, but on GitHub everyone can create its own project(s).

Reply:
It is, although it's more complex on Jira, you need to share the dashboard (with the right group) and you need to share the saved query.